composer.json

```diff
@@ -173,6 +173,12 @@
         "box-compile": [
             "php build/box.phar compile"
         ],
+        "infection-install": [
+            "curl -f -L https://github.com/infection/infection/releases/download/0.26.0/infection.phar -o build/infection.phar"
```

Comment:
Any reason not to pull the latest version always?

Reply:
I'd just prefer to keep the version locked so nothing breaks unexpectedly. Box in particular would break almost every time it updated.

Ultimately we should move this and Box to some other package manager like [Phive](https://phar.io/) so we can manage versions properly.